### PR TITLE
BZ-2138002

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTC 1.7.6
OCP  4.6+
 
Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2138002

Preview: https://53286--docspreview.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/about-mtc.html

### CODE REVIEW

File: modules/migration-understanding-data-copy-methods.adoc [hunk 1]
```diff
@@ -28,6 +28,12 @@ a|* Slower than the snapshot copy method.
 * Optional data verification significantly reduces performance.
 |===
 
+====
+[NOTE]
+Our Restic and Rynsc PV migration assumes that the PVs supported are only volumeMode=filesystem. We *do not* support volumeMode=Block for filesystem migration.
```
Comment (rayfordj) [inline modules/migration-understanding-data-copy-methods.adoc:33]:
```suggestion
Our Restic and Rsync PV migration assumes that the PVs supported are only volumeMode=filesystem. We *do not* support volumeMode=Block for filesystem migration.
```

Comment (CarmiWisemon) [reply modules/migration-understanding-data-copy-methods.adoc]: I corrected the typo. Thanks for noticing it.

**Review by stillalearner — APPROVED** [review]
lgtm

**Comment by bergerhoffer** [comment]
Adding labels as requested by @CarmiWisemon (via email) while they wait for their permissions to add labels themselves.

/label MTC
/label merge-review-needed

**Comment by bergerhoffer** [comment]
@CarmiWisemon We only support as far back as 4.8, so those are the version labels that I'm going to add. Thanks!

**Comment by mburke5678** [comment]
@CarmiWisemon @bergerhoffer Did this PR go through peer review? I am concerned with the volumeMode=filesystem and volumeMode=Block.

s/volumeMode=filesystem/`volumeMode: filesystem`
 s/volumeMode=block/`volumeMode: block` 
 ??

**Comment by bergerhoffer** [comment]
@mburke5678 it may not have. Would you mind doing so during the merge review? I believe this is Carmi's first PR here.

@CarmiWisemon It looks like you should at least put `volumeMode=filesystem`, etc. in backticks. I can't speak to whether they should be changed to colons or not, I think it's probably fine to leave as is if it has had SME/QE review.

Also, the `[NOTE]` line should go above the first `====` line. Like

```
[Note]
====
<Note text here>
====
```

File: modules/migration-understanding-data-copy-methods.adoc [hunk 2]
```diff
@@ -28,6 +28,12 @@ a|* Slower than the snapshot copy method.
 * Optional data verification significantly reduces performance.
 |===
 
+====
+[NOTE]
```
Comment (bergerhoffer) [inline modules/migration-understanding-data-copy-methods.adoc:32]: Move this to above the `====` lines

File: modules/migration-understanding-data-copy-methods.adoc [hunk 3]
```diff
@@ -28,6 +28,12 @@ a|* Slower than the snapshot copy method.
...
+Our Restic and Rsync PV migration assumes that the PVs supported are only volumeMode=filesystem. We *do not* support volumeMode=Block for filesystem migration.
```
Comment (bergerhoffer) [inline modules/migration-understanding-data-copy-methods.adoc:33]: We usually avoid trying to say "our" or "we", can this be reworded? Also, here are a few suggested updates if you agree they are appropriate.

NOTE: Don't push the "Commit suggestion" button you'll see below this :)

```suggestion
The Restic and Rsync PV migration assumes that the PVs supported are only `volumeMode=filesystem`. Using `volumeMode=Block` for file system migration is _not_ supported.
```

**Review by bergerhoffer — COMMENTED** [review]
Adding an official peer review now :)

File: modules/migration-understanding-data-copy-methods.adoc (hunk 2 above)
Comment (CarmiWisemon) [reply modules/migration-understanding-data-copy-methods.adoc:31]: Done

**Pull request merged**